kubemarine/thirdparties.py

```diff
@@ -38,6 +38,29 @@ def enrich_inventory_apply_upgrade_defaults(inventory, cluster):
     return inventory
 
 
+def get_thirdparty_recommended_sha(destination, cluster):
```

Comment:
I think we can add unit tests for this method
* default 3rd-party which requires N.N.N k8s version
* default 3rd-party which requires N.N k8s version
* non-default 3rd-part

Reply:
Tests were added for calculating recommended sha and sha in inventory